### ISSUE TITLE
src/Development/Shake/Language/C/Config.hs:76:12: error: • Couldn't match type ‘RuleResult Config’ with ‘Maybe String’

### ISSUE THREAD

**Issue by juhp** [samplecount/shake-language-c#32]
For Stackage Nightly:

```
src/Development/Shake/Language/C/Config.hs:76:12: error:
    • Couldn't match type ‘RuleResult Config’ with ‘Maybe String’
        arising from a use of ‘addOracle’
    • In a stmt of a 'do' block:
        query <- addOracle
                   $ \ (Config (deps, file, env, key))
                       -> Map.lookup key <$> fileCache (deps, file, env)
      In the expression:
        do fileCache <- newCache $ \ (deps, file, env) -> do ...
           query <- addOracle
                      $ \ (Config (deps, file, env, key))
                          -> Map.lookup key <$> fileCache (deps, file, env)
           return
             $ \ deps file env key -> query (Config (deps, file, env, key))
      In an equation for ‘mkConfig’:
          mkConfig
            = do fileCache <- newCache $ \ (deps, file, env) -> ...
                 query <- addOracle
                            $ \ (Config (deps, file, env, key))
                                -> Map.lookup key <$> fileCache ...
                 return $ \ deps file env key -> query (Config ...)
   |
76 |   query <- addOracle $ \(Config (deps, file, env, key)) -> Map.lookup key <$> fileCache (deps, file, env)
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

**Comment by juhp** [comment]
I guess it is due to shake-0.16

**Comment by kaoskorobase** [comment]
Thanks for the report, will try to fix ASAP and ping you again.

**Comment by kaoskorobase** [comment]
Fixed in [version 0.11.0](https://hackage.haskell.org/package/shake-language-c-0.11.0). @juhp will you re-enable `shake-language-c` in stack or do I need to send a PR? Thanks!

**Comment by bergmark** [comment]
I re-enabled it, but these comments can easily get lost so the best bet is to send a PR.
